software/patches/buildroot/latmem.patch

```diff
@@ -0,0 +1,216 @@
+From 31e5e2bec81655ca65886a79728290d56b8bc840 Mon Sep 17 00:00:00 2001
+From: Tu Dang <tu.dang@wdc.com>
+Date: Mon, 16 Dec 2019 17:01:17 -0800
+Subject: [PATCH 2/2] Added latmem package
+
+---
+ package/Config.in           |   1 +
+ package/latmem/Config.in    |   4 ++
+ package/latmem/latmem.mk    |  19 ++++++
+ package/latmem/src/Makefile |  17 +++++
+ package/latmem/src/latmem.c | 120 ++++++++++++++++++++++++++++++++++++
+ 5 files changed, 161 insertions(+)
+ create mode 100644 package/latmem/Config.in
+ create mode 100644 package/latmem/latmem.mk
+ create mode 100644 package/latmem/src/Makefile
+ create mode 100644 package/latmem/src/latmem.c
+
+diff --git a/package/Config.in b/package/Config.in
+index f72c77b416..e4a79ece5a 100644
+--- a/package/Config.in
++++ b/package/Config.in
+@@ -1871,6 +1871,7 @@ menu "Miscellaneous"
+ 	source "package/gr-osmosdr/Config.in"
+ 	source "package/gsettings-desktop-schemas/Config.in"
+ 	source "package/haveged/Config.in"
++	source "package/latmem/Config.in"
+ 	source "package/linux-syscall-support/Config.in"
+ 	source "package/mcrypt/Config.in"
+ 	source "package/mobile-broadband-provider-info/Config.in"
+diff --git a/package/latmem/Config.in b/package/latmem/Config.in
+new file mode 100644
+index 0000000000..6530b7ce3b
+--- /dev/null
++++ b/package/latmem/Config.in
+@@ -0,0 +1,4 @@
++config BR2_PACKAGE_LATMEM
++	bool "latmem"
++	help
++		Simple remote socket's memory test
+diff --git a/package/latmem/latmem.mk b/package/latmem/latmem.mk
+new file mode 100644
+index 0000000000..1f3b44dbe5
+--- /dev/null
++++ b/package/latmem/latmem.mk
+@@ -0,0 +1,19 @@
++################################################################################
++#
++# latmem
++#
++################################################################################
++
++LATMEM_VERSION = 1.0
++LATMEM_SITE = ./package/latmem/src
++LATMEM_SITE_METHOD = local
++
++define LATMEM_BUILD_CMDS
++    $(MAKE) CC="$(TARGET_CC)" LD="$(TARGET_LD)" -C $(@D)
++endef
++
++define LATMEM_INSTALL_TARGET_CMDS
++    $(INSTALL) -D -m 0755 $(@D)/latmem $(TARGET_DIR)/usr/bin
++endef
++
++$(eval $(generic-package))
+diff --git a/package/latmem/src/Makefile b/package/latmem/src/Makefile
+new file mode 100644
+index 0000000000..94ca525859
+--- /dev/null
++++ b/package/latmem/src/Makefile
+@@ -0,0 +1,17 @@
++export PATH:=$(HOME)/Workspace/riscv64--glibc--bleeding-edge-2018.11-1/bin:$(PATH)
++CC=riscv64-linux-
```

Comment:
I don't think you need to explicitly export PATH or CC for individual package. Buildroot should take care of that.
It also may not work on other machines as toolchain prefix or toolchain install path may be different.